daemon/events.go

```diff
@@ -80,6 +80,29 @@ func (daemon *Daemon) LogNetworkEventWithAttributes(nw libnetwork.Network, actio
 	daemon.EventsService.Log(action, events.NetworkEventType, actor)
 }
 
+// LogDaemonEventWithAttributes generates an event related to the daemon itself with specific given attributes.
+func (daemon *Daemon) LogDaemonEventWithAttributes(action string, attributes map[string]string) {
+	if daemon.EventsService != nil {
+		for _, label := range daemon.configStore.Labels {
```

Comment:
Delete this since we're already marshalling `daemon.configStore.Labels` during reload, no?